### ISSUE TITLE
CurlMollieHttpAdapter.php in mollie-api-php-2.76.0.zip

### ISSUE THREAD

**Issue by jwvanooijen** [mollie/mollie-api-php#757]
## Specifications

  - API Version: 2.76.0

## Describe the issue

The 'use' statement for 'CaBundle' in the file _CurlMollieHttpAdapter.php_ in _mollie-api-php-2.76.0.zip_ reads this:

`use Composer\CaBundle\CaBundle;`

but it should be this:

`use _PhpScoper506056c89102\Composer\CaBundle\CaBundle;`

The same file in _mollie-api-php.zip_ is correct.

**Comment by sandervanhooft** [comment]
Thanks for reporting @jwvanooijen , I am investigating the issue.

**Comment by sandervanhooft** [comment]
Hi @jwvanooijen,

I have some trouble confirming the issue. Are you by any chance looking at a different download or asset?

I went to https://github.com/mollie/mollie-api-php/releases/tag/v2.76.0, then downloaded `mollie-api-php.zip` from here:

<img width="201" alt="image" src="https://github.com/user-attachments/assets/ffac0c82-b6b9-4168-9fe3-314117677119" />

After unpacking I looked up the file `src/HttpAdapter/CurlMollieHttpAdapter.php`, the import statement I am seeing there is:

```php
use _PhpScoper506056c89102\Composer\CaBundle\CaBundle;
```

<img width="767" alt="image" src="https://github.com/user-attachments/assets/b06722d9-04fb-4e19-8617-2e8420978ffb" />

Let me know if you feel I missed anything.

**Comment by sandervanhooft** [comment]
Hi @jwvanooijen ,

Thanks for clearing that up. The behavior you're describing actually is by design.

- The 2nd and 3rd in the list of Assets are the default packaged solutions, without Scoper applied.
- The 1st is a scoped version using Scoper, which prevents colliding installations of popular packages. It's a workaround specifically for collision issues we've encountered in the past, i.e. where Guzzle was already present in the system. You can see the zip gets generated here:
https://github.com/mollie/mollie-api-php/blob/b6e9848e4893d1d306fba54da3ac6f4a31a40368/.github/workflows/createzip.yml#L19-L39

In any case, if your framework allows it, I strongly recommend to use `composer` for managing dependencies.

**Comment by jwvanooijen** [comment]
Hello Sander,

thank you. I prefer not to use _composer_, I am trying to keep the website source code as light-weight as possible. There are no issues with dependencies on my website, the _mollie-api_ is the only external package.

So I am using the source from the 2nd Asset, but have to get the vendor part of the 1st Asset in order to get the _autoload_ function. From _vendor\autoload.php_ the code path presumably ends up in _autoload-static.php_. Here, all three references to _Composer\\CaBundle\\_ are prepended with `_PhpScoper506056c89102\\`. The file _CurlMollieHttpAdapter.php_ from the 2nd Asset has the use statement mentioned earlier: `use Composer\CaBundle\CaBundle;`, which lacks the prepended `_PhpScoper506056c89102\\`, resulting in errors.
I could solve the errors by prepending the `_PhpScoper506056c89102\\` to the `use` statement.

Possibly a better solution would be to remove the three instances of `_PhpScoper506056c89102\\` in _autoload-static.php_ and the ones in _CaBundle.php_ and _autoload_classmap.php_.

In short, I wish to use the _mollie-api_ with a simple _autoload_-funtion without _composer_.
Thanks again,
Johan

**Comment by sandervanhooft** [comment]
Hi @jwvanooijen ,

I have just tested this locally, and did not run into any issues.

Using the first zip asset in the list, I've unpacked it and added a sample file (contents below). After updating the test key it worked. I haven't used composer. Feel free to give it a try.

```php
<?php

namespace _PhpScoper506056c89102;

/*
 * Example: How to fetch a page of payments using the Mollie API
 */

// Make sure to disable the display of errors in production code!
ini_set('display_errors', '1');
ini_set('display_startup_errors', '1');
error_reporting(E_ALL);

// Require the autoloader to handle dependencies
require_once __DIR__ . "/vendor/autoload.php";

// Initialize the Mollie API library with your API key
$mollie = new \Mollie\Api\MollieApiClient();
$mollie->setApiKey("your_api_key"); // Replace with your actual API key

try {
    // Get the first page of payments for this API key, ordered by newest first
    $payments = $mollie->payments->page();
    
    // Output the results
    echo "Total number of payments: " . $payments->count() . "\n\n";
    
    foreach ($payments as $payment) {
        echo "Payment ID: " . $payment->id . "\n";
        echo "Description: " . $payment->description . "\n";
        echo "Amount: " . $payment->amount->currency . " " . $payment->amount->value . "\n";
        echo "Status: " . $payment->status . "\n";
        echo "Created At: " . $payment->createdAt . "\n";
        echo str_repeat("-", 50) . "\n";
    }
    
} catch (\Mollie\Api\Exceptions\ApiException $e) {
    echo "API call failed: " . htmlspecialchars($e->getMessage());
}
```

**Comment by jwvanooijen** [comment]
hi @sandervanhooft,
thank you. It also works with the second zip asset, on the condition that the `use` statement in _CurlMollieHttpAdapter.php_ has `_PhpScoper506056c89102\` prepended as mentioned before; otherwise you get a `Fatal error:  Uncaught Error: Class 'Composer\CaBundle\CaBundle' not found in /.../mollie/src/HttpAdapter/CurlMollieHttpAdapter.php:81.`.

What is the purpose of this second zip asset?
